scripts/dev-template.mjs

```diff
@@ -18,6 +18,9 @@ import { $ } from 'zx'
 import fs from 'fs/promises'
 import path from 'path'
 
+// eslint-disable-next-line functional/immutable-data
+process.env.FORCE_COLOR = '1'
```

Comment:
When I ran commands like `yarn dev:vue3`, `dev-template.mjs` runs the template, but it didn't display any color at all. I'm not too sure but I think there is some issue regarding color + subprocess.

https://google.github.io/zx/known-issues#colors-in-subprocess